### ISSUE TITLE
Request for Sample Projects

### ISSUE THREAD

**Issue by hashg** [getoutreach/epf#28]
Please publish , TODO & Blog examples ported to epf. Also please include a example involving 3level nested route.

**Comment by ghempton** [comment]
Eventually we will have some better official sample projects. For now, you can check out this project by @rayb : https://github.com/rayb/epf_skeleton.

**Comment by kiwiupover** [comment]
I have forked the ember_data_example and replaced ember_data with EPF at [epf_example](https://github.com/kiwiupover/epf_example). 

It is a simple contact management app. It has three levels of models.

I have added a contact group with I think shows off the power of EPF in that you are able to create a group and a contact and persist them both on save.

**Comment by g13013** [comment]
@ghempton epf must have a doc also, it's one of the big obstacles using it.

**Comment by ghempton** [comment]
Yeah I would like to flesh out the inline docs in the code and add a docs section to the site. Also on the roadmap is a guides section.

**Comment by g13013** [comment]
As i worked a lot with extjs, I suggest using [jsduck](https://github.com/senchalabs/jsduck) to document epf, it was designed to be used with Extjs, but it works also with a list of tags witch some of them are already used in epf, it generates a pretty clear api [doc](http://docs.sencha.com/extjs/4.2.1/#!/api)!
